src/client/pps/pps.proto

```diff
@@ -196,6 +201,7 @@ message CreateJobRequest {
   repeated JobInput inputs = 4;
   Job parent_job = 5;
   bool force = 6;
+  Service service = 8;
```

Comment:
I'd put a comment here that the presence of this field changes the behavior of jobs